frontend/app/forms/MemberForm.scala

```diff
@@ -101,6 +105,16 @@ object MemberForm {
 
   case class FeedbackForm(category: String, page: String, feedback: String, name: String, email: String)
 
+  val abTestFormatter: Formatter[JsValue] = new Formatter[JsValue] {
+    override def bind(key: String, data: Map[String, String]): Either[Seq[FormError],JsValue] = {
+      printf(key)
+      val parse: JsValue = Json.parse(URLDecoder.decode(data(key),"UTF8"))
```

Comment:
Maybe we could use a constant for UTF-8

http://stackoverflow.com/questions/6698354/where-to-get-utf-8-string-literal-in-java